### ISSUE TITLE
How to fix error  : The SDK 'Microsoft.NET.Sdk' specified could not be found. 

### ISSUE THREAD

**Issue by saaberw** [saaberw/DotnetCore#2]
You will get "the current .NET SDK does not support targeting .NET Core 3.0" instead of the project building, even if you have the SDK installed, even if you have both x64 and x86 installed.

Adding MSBuildSdksPath in your Environment User Variables that points to "C:\Program Files\dotnet\sdk" fixes the problem.

The installer should add this environment user variable if it's necessary.

I encountered this error after playing around with .Net Core 6.0 installation and seemingly messing it up.

To fix the problem, I manually set the **MSBuildSDKsPath** environment variable to point to 6.0.0's SDK path, which, for me with x64, this was at: **C:\Program Files\dotnet\sdk\2.0.0\Sdks** .

Adding MSBuildSdksPath in your Environment User Variables that points to "C:\Program Files\dotnet\sdk" fixes the problem.

The installer should add this environment user variable if it's necessary.

First, Run SystemPropertiesAdvanced.exe
Go : **Advanced** tab > Click **"Environment Variables"**

on "System variables" click New..
and fill that like this:

![image](https://github.com/saaberw/DotnetCore/assets/12484563/0ea26eca-acb1-4e94-9945-8cda442aba3a)

Basically, if you have Sdk="Microsoft.NET.Sdk" in your .csproj, then a folder with the same name should exist at your MSBuildSDKsPath location.
